content/applications/services/field_service/default_warehouse.rst

```diff
@@ -16,15 +16,15 @@ Configuration
 =============
 
 To set up a user default warehouse, the :doc:`storage locations
-</applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations>`
+<../../inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations>`
 feature needs to be activated in the **Inventory** app. It is also necessary to have more than one
```

Comment:
```suggestion
feature needs to be activated in the *Inventory* app. It is also necessary to have more than one
```

Reply:
Same as above-- since this is a field service doc, I plan to not make any content changes, and just wanted to update a link that wouldve been broken upon publishing this PR.